packages/examples/packages/notifications/src/index.tsx

```diff
@@ -39,6 +40,28 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
         },
       });
 
+    case 'inApp-expanded':
```

Comment:
Manifest will need to be updated to make CI pass 🫡

Reply:
I was actually working on adding `snaps-jest` support so that's why I didn't get around to updating the manifest yet 😉